pkg/specgen/specgen.go

```diff
@@ -83,6 +83,11 @@ type ContainerBasicConfig struct {
 	// instead.
 	// Optional.
 	StopTimeout *uint `json:"stop_timeout,omitempty"`
+	// Timeout is a maximum time in seconds the container will run before
+	// main process is sent SIGKILL.
```

Comment:
So is this immediate SIGKILL, or does it use the normal stop logic (SIGTERM -> timeout -> SIGKILL)?

Reply:
We are using the conmon --timeout flag, which is defined as

       -T, --timeout Kill container after specified timeout in seconds.

Which to me means the KILL signal gets sent.  I don't think we should make it more complicated then this.  If users want more complicated they should do it via bash.